2996_reflection/reflection.md

```diff
@@ -5529,7 +5739,7 @@ consteval bool is_static_member(info r);
 consteval bool is_base(info r);
 ```
 
-[#]{.pnum} *Returns*: `true` if `r` represents a class member, namespace member, non-static data member, static member, base class specifier, respectively. Otherwise, `false`.
+[#]{.pnum} *Returns*: `true` if `r` represents a class member, namespace member, non-static data member, static member, direct base class relationship, respectively. Otherwise, `false`.
```

Comment:
Pre-existing, but this is missing an "or"

Reply:
Good catch, done.